Core/src/org/sleuthkit/autopsy/datamodel/accounts/Accounts.java

```diff
@@ -580,11 +580,9 @@ protected boolean createKeys(List<Long> list) {
                     + getRejectedArtifactFilterClause(); //NON-NLS
             try (SleuthkitCase.CaseDbQuery results = skCase.executeQuery(query);
                     ResultSet rs = results.getResultSet();) {
-                List<Long> tempList = new ArrayList<>();
                 while (rs.next()) {
-                    tempList.add(rs.getLong("artifact_obj_id")); // NON-NLS
+                    list.add(BlackboardArtifactNode.createNodeKey(skCase.getBlackboard().getDataArtifactById(rs.getLong("artifact_obj_id"))));
```

Comment:
There is actually a bug here that we should just quickly fix while we are here. The NetBeans infrastructure is listening to the list passed in to createKeys() and doing work every time a key is added to the list. We should collect up all of our node keys and then add them to the output list in one go.